### DISCUSSION TITLE
Como gerar o certificado para autenticação?

### DISCUSSION THREAD

**Discussion by anhaabaete** [andreroggeri/pynubank#368] [Q&A] [answered]
Olá galera,
tinha o certificado na minha máquina e ele expirou dia 28/11
Lembro de ter procurado na internet e achado fácil como gera esse certificado. Procurei agora e não acho de forma alguma.
Também tentei fazer login com o QrCode e deu problema.

Obrigado

**Comment by andreroggeri** [comment, marked as answer]
Se você estiver com o pynubank instalado basta rodar o comando `pynubank` no terminal que ele guiará você na geração do certificado

**Comment by msoffo** [comment]
Olá comunidade,
Eu não estou conseguindo gerar o certificado, mesmo inserindo os dados corretos.
Pelo metodo QrCode e pela web e consigo sem problemas
Será que mudou algum procedimento?

$ pynubank
[*] Starting PyNubank context creation.
[*] Generated random id: vwa1w0abvl33
[>] Enter your CPF(Numbers only): ***********
[>] Enter your password (Used on the app/website): 
[*] Requesting e-mail code
[*] Failed to request code. Check your credentials!

**Reply by andreroggeri** [reply]
Você está na versão mais recente ?

Houve uma mudança na API do Nubank que quebrou as versões antigas da biblioteca

**Reply by msoffo** [reply]
Obrigado pela pronta resposta, @andreroggeri 🙏 

Antes de fazer a pergunta, eu tinha certeza que estava na ultima versão, mas agora que você falou, percebei que não tinha tanta certeza assim... 😕 

Atualizei e deu certo. Valeu, irmão! 🤜🤛
